src/components/DefineCompany/OfficeAddresses.vue

```diff
@@ -474,7 +473,7 @@ export default class OfficeAddresses extends Mixins(CommonMixin, EntityFilterMix
 
   /** Emits updated addresses object to the parent page. */
   @Emit('update:addresses')
-  private emitAddresses (): object {
```

Comment:
Actually, it's not an object that is returned, since there are some missing properties/methods that make up an actual object.